docs/vector-indexes.md

```diff
@@ -72,11 +72,58 @@ from wagtail_vector_index.index.models import EmbeddableFieldsVectorIndex
 
 class MyEmbeddableFieldsVectorIndex(EmbeddableFieldsVectorIndex):
     querysets = [
-        MyModel.objects.all(),
-        MyOtherModel.objects.filter(name__startswith="AI: "),
+        InformationPage.objects.all(),
+        BlogPage.objects.filter(name__startswith="AI: "),
     ]
 ```
 
+We're hoping to handle this automatically in the near future. But for now, you'll also need to override a couple of methods on your class to allow it to succesfully retrieve content for indexing. For example:
+
+```python
+from wagtail.models import Page
+
+
+class MyEmbeddableFieldsVectorIndex(EmbeddableFieldsVectorIndex):
+    ...
+
+    def get_converter(self, model_class=None):
+        return self.get_converter_class()(model_class or Page)
```

Comment:
This should do for the sake of the example, but natively, it would be great to identify the base concrete model automatically here